src/hw/pcie_cfg.cpp

```diff
@@ -27,6 +27,17 @@ static constexpr uint8_t pcieBifurcateXXX8XXX8 = 3;
 static constexpr uint8_t pcieBifurcateXXXXXX16 = 4;
 static constexpr uint8_t pcieBifurcateXXXXXXXX = 0xF;
 
+static constexpr uint8_t bmcPcieBifurcateX4X4X4X4 = 0x44;
+static constexpr uint8_t bmcPcieBifurcateX4X4XXX8 = 0x48;
+static constexpr uint8_t bmcPcieBifurcateXXX8X4X4 = 0x84;
+static constexpr uint8_t bmcPcieBifurcateXXX8XXX8 = 0x88;
+static constexpr uint8_t bmcPcieBifurcateXXXXXX16 = 0x16;
+static constexpr uint8_t bmcPcieBifurcate____X4X4 = 0xF4;
+static constexpr uint8_t bmcPcieBifurcate____XXX8 = 0xF8;
+static constexpr uint8_t bmcPcieBifurcateX4X4____ = 0x4F;
+static constexpr uint8_t bmcPcieBifurcateXXX8____ = 0x8F;
+static constexpr uint8_t bmcPcieBifurcateXXXXXXXX = 0xFF;
```

Comment:
Looks a bit inconsistent. If you use underscores above for 'unchanged' bifurcation, then why use `X` here ?
Maybe call this one `bmcPcieBifurcateUnchanged` ? Or call the above ones `...BifurcateXXXXX4X4`, `...BifurcateX4X4XXXX` and so on?